TestCases/cont_adj_euler/wedge/inv_wedge_ROE_multiobj.cfg

```diff
@@ -362,7 +377,9 @@ VISUALIZE_DEFORMATION= YES
 %
 % Optimization objective function with scaling factor
 % ex= Objective * Scale
-OPT_OBJECTIVE=LIFT*-1.0E4;OUTFLOW_GENERALIZED*0.01 
+OPT_OBJECTIVE=AVG_TOTAL_PRESSURE*1E-8;(DRAG = 0.045)*1E4
```

Comment:
In other words, the goal is to be able to cut-and-paste directly from the OPT_CONSTRAINT line in order to switch between using penalty functions inside the objective vs using the optimization algorithm.